packages/ui/src/components/vuestic-components/va-form/VaForm.vue

```diff
@@ -8,10 +8,10 @@
 </template>
 
 <script lang="ts">
-import { inject, provide } from 'vue'
+import { provide } from 'vue'
 import { Options, mixins, prop, Vue, setup } from 'vue-class-component'
-import { FormComponentMixin } from '../../vuestic-mixins/FormComponent/FormComponentMixin'
-import { FormProvider, FormServiceKey } from './consts'
+import { useFormProvider } from '../../vuestic-mixins/FormComponent/cFormComponent'
```

Comment:
Can we not use abbreviations?
`FormComponentComposable`?

Reply:
Probably it's better to have consistent naming everywhere in the project and name it the same way as Vue documentation suggests:

- " **useFormComponent** " instead of " **cFormComponent** "